docs/coverage-frequency.md

```diff
@@ -0,0 +1,86 @@
+4. The last part of the query (actually, the first syntax-wise), is getting the actual rate with [the `rate` function](https://prometheus.io/docs/prometheus/latest/querying/functions/#rate), which simply returns the per-second average rate at each point in time, taking into account the vector of values we have. In this case, it gets the per-second average rate of the last two hours at each point in time. We finally apply [the `sum` operator](https://prometheus.io/docs/prometheus/latest/querying/operators/#aggregation-operators), which simply aggregates all the labels to get a single value per point in time.
+
+Now that we understand the first line, which gets us the average rate of requests to `upsertDraft` in a two-hour window, we can understand the whole thing. We do exactly the same with `createPost`, so we get the rate to `createPost` in the same two-hour window, and then we divide one between the other. What does that mean? What we obtain is the ratio of rates between these two endpoints; i.e., how they relate to one another: If at a point in time we get that this ratio is `2.0`, that means that the server received two times more requests for `upsertDraft` than for `createPost` in the previous two-hour window. If the value is `0.5`, that means that it's `createPost` the endpoint that is hit double the times than `upsertDraft`. If we draw this line in our time range, we get something like the following:
+
+![Ratio graph](https://community.mattermost.com/files/nrr9zp4a87rp9x8dx66eej961e/public?h=vA0pV04TBVn7mS3wLCzvLxl-_SANGS6tx4i4hOJO5fs)
+
+Now we know, for each point in time between 16:00 and 18:00, the ratio between the rates of `upsertDraft` and `createPost`. But remember that we chose a two-hour window with the `[2h]` range vector selector. So if we look at the time 16:30, we're actually considering data from 14:30 to 16:30. We're mostly interested in the 16:00-18:00 window, so we are actually interested in a single point from this line: the very last one.
+
+We can inspect it by hovering over the last part of the line or by scrolling down and looking at the table below the graph, which conveniently lists the very last value, at 18:00 UTC. In this case, `0.452`.
+
+![Ratio graph - value](https://community.mattermost.com/files/4rnwfgaq7idc9rtm45rye7f44w/public?h=UjBdpoIYymgbjLzgCHA6HIFofDBlIznIMEqpHia0GC0)
+
+Given this ratio, and as the frequency of the `createPost` action is [already configured to be 1.5](https://github.com/mattermost/mattermost-load-test-ng/blob/master/loadtest/control/simulcontroller/controller.go#L125-L128), we can simply multiply these two numbers to get the frequency of the new action:
+
+```
+1.5 * 0.452 = 0.678
+```
+
+When adding our new action, the `frequency` should be set to `0.678`. Done!
+
+## Summary
```

Comment:
Something to consider: move the summary to the top of the page as a type of TL;DR. Doing so can help readers quickly confirm whether this is the page of content they need by reviewing the summary.

Reply:
I thought about this, but wasn't sure it was a good idea. Thanks for confirming my early, then discarded, thoughts!